standalone/detectors/fasttrack/include/PrefixTree_StackDepot.h

```diff
@@ -0,0 +1,95 @@
+#ifndef PREFIXTREE_STACKDEPOT_HEADER_H
+#define PREFIXTREE_STACKDEPOT_HEADER_H 1
+#pragma once
```

Comment:
Please remove insignificant `1` and the redundant pragma since include guards already exist and are used 
```suggestion
#define PREFIXTREE_STACKDEPOT_HEADER_H
```